src/main/resources/application.conf

```diff
@@ -0,0 +1,18 @@
+application {
```

Comment:
Not important here, but we shouldn't be checking in config into repository.
The example-application.conf should have structure, but with dummy values.
Whilst Application.conf should be added to gitignore file and each of us would copy example file and rename to application.conf and use this without risk of checking in passwords.